### PR TITLE
[NOT TO BE MERGED, experimental branch] [WIP] moocorrel

### PR DESCRIPTION
**Pull request by teytaud**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] The documentation is up-to-date with the changes I made.
- [ ] I have read the **CONTRIBUTING** document and completed the CLA (see **CONTRIBUTING**).
- [ ] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: nevergrad/functions/multiobjective/core.py [hunk 1]
```diff
@@ -110,8 +110,9 @@ def pareto_front(self, size: Optional[int] = None, subset: str = "random") -> Li
             return random.sample([p[0] for p in self._points], size)
         possibilities: List[Any] = []
         scores : List[float] = []
-        for u in range(30):
-            possibilities += [random.sample(self._points, size)]
+        localrandom = random.Random(0)
```
Comment (jrapin) [inline nevergrad/functions/multiobjective/core.py]: Seeding is never a good idea! random.Random() should work,
Once we store parameters, we can use the random number generator in it

File: nevergrad/functions/multiobjective/core.py [hunk 2]
```diff
@@ -127,4 +128,4 @@ def pareto_front(self, size: Optional[int] = None, subset: str = "random") -> Li
                             raise ValueError(f'Unknown subset for Pareto-Set subsampling: "{subset}"')
                     score += best_score ** 2
                 scores += [score]
-        return [p[0] for p in possibilities[scores.index(min(scores))]]
+        return [p[0] for p in possibilities[scores.index(min(scores))]], scores
```
Comment (jrapin) [inline nevergrad/functions/multiobjective/core.py]: Do we really need the scores? That makes the API difficult to use. If we do we need another solution

**Comment by jrapin** [comment]
Also can you use a better name for the PR, it would help a lot to follow what happens in the PRs, and help a lot as well if they get merged to have a feeling of the changes that have been made

**Comment by teytaud** [comment]
merged differently and elsewhere

**Pull request closed without merging**